types/evidence.go

```diff
@@ -32,17 +35,27 @@ type Evidence interface {
 type DuplicateVoteEvidence struct {
 	VoteA *Vote `json:"vote_a"`
 	VoteB *Vote `json:"vote_b"`
+
+	// abci specific information
```

Comment:
can we explain what each field is?

Reply:
do you mean just these new fields that I have added or all of them?

Reply:
new fields. voteA and voteB are pretty explanatory